sources/code/main/windows/main.ts

```diff
@@ -26,13 +26,65 @@ import { commonCatches } from "../modules/error";
 
 import type { PartialRecursive } from "../../common/global";
 import { nativeImage } from "electron/common";
+// eslint-disable-next-line
+// @ts-ignore - This will ignore the error if pipewire isn't installed
+import type { PipewireLink, PipewireNode, PipewirePort } from "node-pipewire/build/types";
+// eslint-disable-next-line
+// @ts-ignore - will also ignore
+import type PipewireModule from "node-pipewire";
+
+// eslint-disable-next-line
+const pw: typeof PipewireModule | null = (() => { try {
+  // eslint-disable-next-line
+  return require("node-pipewire") as typeof PipewireModule;
+} catch(e) {
+  console.log(e);
+  return null;
+}})();
 
 interface MainWindowFlags {
   startHidden: boolean;
   screenShareAudio: boolean;
 }
 
+interface AudioInformation {
+  selectedAudioNodes: string[] | null;
+}
+
+const blacklistInputNodes: number[] = [];
+
+// eslint-disable-next-line import/no-unused-modules
```

Comment:
> You should avoid using eslint-disable for all of the rules and lines there and explain why you disable the linter. I won't allow bypassing the problems – you should resolve the linter rules or explain why you didn't for each.

If module is unneeded, there's a reason for that. Don't export it or make it actually be needed somewhere!